add intro for explicit contract disclosure (#360)
digital-asset/docs.daml.com: docs/2.7.0/docs/app-dev/explicit-contract-disclosure.rst

> Explicit Contract Disclosure (Experimental)

Reviewer: ```suggestion
Explicit Contract Disclosure (Alpha)
```

"Experimental" Does not have a clear meaning. Use the official status labels.

###########################################

> In Daml, you must specify upfront who can view data. To change who can see the data, you would typically need to rewrite the asset with a new annotation. Canton 2.7 introduces explicit contract disclosure as a feature that allows you to seamlessly delegate contract read rights to a non-stakeholder using off-ledger data distribution. This supports efficient, scalable data sharing on the ledger. 

Reviewer 2: Can we specify maybe a bit better what **annotation** might refer to?

> In Daml, you must specify upfront who can view data. To change who can see the data, you would typically need to rewrite the asset with a new annotation. Canton 2.7 introduces explicit contract disclosure as a feature that allows you to seamlessly delegate contract read rights to a non-stakeholder using off-ledger data distribution. This supports efficient, scalable data sharing on the ledger. 

Reviewer: ```suggestion
In Daml, you must specify upfront who can view data using `observer` annotations on contracts. To change who can see the data, you would typically need to rewrite the contract (eg an asset) with a new annotation. Canton 2.7 introduces explicit contract disclosure as a feature that allows you to seamlessly delegate contract read rights to a non-stakeholder using off-ledger data distribution. This supports efficient, scalable data sharing on the ledger. 
```

> Canton 2.7 introduces explicit contract disclosure as a feature that allows you to seamlessly delegate contract read rights to a non-stakeholder using off-ledger data distribution. This supports data sharing on the ledger in a scalable manner. In Daml, you must specify upfront who can view data. To change who can see the data, you would typically need to rewrite the asset with a new annotation. Explicit contract disclosure provides a more efficient method.

Reviewer 3: ```suggestion
In Daml, you must specify upfront who can view data. To change who can see the data, you would typically need to rewrite the asset with a new annotation. Canton 2.7 introduces explicit contract disclosure as a feature that allows you to seamlessly delegate contract read rights to a non-stakeholder using off-ledger data distribution. This supports efficient, scalable data sharing on the ledger. 
```

Here are some use cases that illustrate how you might benefit from explicit contract disclosure:

> - You want to provide proof of the price data for a stock transaction. Instead of subscribing to price updates and being inundated with possibly 1,000 price updates every minute, you could put the price data on a separate ledger that has a Web 2.0 API. You can then use that API to feed only the current price back into the ledger once.

Reviewer 3: ```suggestion
- You want to provide proof of the price data for a stock transaction. Instead of subscribing to price updates and potentially being inundated with thousands of price updates every minute, you could put the price data on a separate ledger that has a Web 2.0 API. You can then use that API to feed only the current price back into the ledger once.
```

> - You need to distribute software updates for an app. In an open network, that could mean shipping the update to millions of users. Instead, you can run automations centrally and use data from the user to build a transaction.

Reviewer 2: > Instead, you can run automations centrally and use data from the user to build a transaction.

I don't really understand this sentence: which data can **you** use and who is the **user**?

> - You need to distribute software updates for an app. In an open network, that could mean shipping the update to millions of users. Instead, you can run automations centrally and use data from the user to build a transaction.

Reviewer: ```suggestion
- You need to distribute software updates for an app. In an open network, that could mean shipping the update to millions of users. Instead, you can run automations centrally and use data from the user to build a transaction.
- You want to run an open market on ledger. Rather than making all bids and asks explicitly visible to all marketplace users, you serve market data though standard Web 2.0 APIs. At the point of use, the available bids and asks are fed back into the transactions to get the same activeness and correctness guarantees that would be provided had they been shared though the observer mechanism.
```

> - You need to distribute software updates for an app. In an open network, that could mean shipping the update to 1 million users. Instead, you can run automations centrally and use data from the user to build a transaction.

Reviewer 3: ```suggestion
- You need to distribute software updates for an app. In an open network, that could mean shipping the update to millions of users. Instead, you can run automations centrally and use data from the user to build a transaction.
```
